src/runtime/virtcontainers/api.go

```diff
@@ -137,6 +138,15 @@ func CleanupContainer(ctx context.Context, sandboxID, containerID string, force
 
 	s, err := fetchSandbox(ctx, sandboxID)
 	if err != nil {
+		if os.IsNotExist(err) {
+			// the sandbox store may be destroyed
+			// when the monitor watches that sandbox stopped unexpectedly
```

Comment:
Is this only valid for the case monitor watching ?

Reply:
yes,  from the situations I have encountered， it looks like this.

Reply:
Yeah, I think it's better to comment it in a generic way, not only for monitor watch event.
Just explaining that the sandbox store has been destroyed or the sandbox doesn't exist is fine.

Reply:
oh, this comment means that when we got here, the sandbox may have been destroyed from the store when handling an unexpectedly exiting. 
other situations,  wouldn’t be here.